docker-compose.yaml

```diff
@@ -13,8 +13,9 @@ services:
       LOGIN_REQUIRED: ${LOGIN_REQUIRED:-false}
       AUTH_USERNAME: ${AUTH_USERNAME:-grants}
       AUTH_PASSWORD_HASH: ${AUTH_PASSWORD_HASH:-grants}
-      GOOGLE_TAG_MANAGER_KEY: 'GTM-WJ5C78H'
-      GOOGLE_TAG_MANAGER_SERVER_KEY: 'UA-179628664-4'
+      GOOGLE_TAG_MANAGER_KEY: 'GT-MJPRKW7'
+      GOOGLE_TAG_MANAGER_SERVER_KEY: 'NVW29SmFTueM1qVawk2H2Q'
```

Comment:
in docker-compose file, GOOGLE_TAG_MANAGER_KEY is incorrect, and GOOGLE_TAG_MANAGER_SERVER_KEY should come from ${GOOGLE_TAG_MANAGER_SERVER_KEY:-notset} (aka the values.yaml and config-map.yaml files). Same with ANALYTICS_PROPERTY_API

The key is also incorrect according to calf housing :)

Reply:
From calf housing docker-compose.yaml file:
![image](https://github.com/DEFRA/ffc-grants-slurry-web/assets/15157025/1eea6486-6239-45bf-9457-7dd6f849afe4)

Basically we shouldn't be creating these keys in the file, we should be referencing the environment variables instead :)